frontend/src/views/Docs.jsx

```diff
@@ -98,29 +104,34 @@ const Docs = (defaultprops) => {
     setAnchorEl(null);
   }
 
+  const handleCollapse = (index) => {
+    setOpen(isopen === index ? -1 : index)
+  };
+
   const SidebarPaperStyle = {
     backgroundColor: theme.palette.surfaceColor,
     overflowX: "hidden",
     position: "relative",
-    padding: 30,
+    paddingLeft: 15,
+    paddingRight: 15,
     paddingTop: 15,
     marginTop: 15,
-		minHeight: "80vh",
+    minHeight: "80vh",
     //height: "50vh",
   };
 
   const SideBar = {
-    minWidth: 250,
-    maxWidth: 300,
-		left: 0,
-		position: "sticky",
-		top: 50,
-		minHeight: "90vh",
-		maxHeight: "90vh",
-		overflowX: "hidden",
-		overflowY: "auto",
-		zIndex: 1000,
-		//borderRight: "1px solid rgba(255,255,255,0.3)",
+    minWidth: 300,
+    width: "20%",
```

Comment:
Is this 20% of screen or 20% of parent?

Reply:
> Is this 20% of screen or 20% of parent?

Of parent which is Body which covers an entire screen i guess